lib/constructorio.js

```diff
@@ -32,6 +32,27 @@ function handleServerResponse(error, response, callback) {
   }
 }
 
+/**
+ * @description Make requests to the catalog endpoint
+ */
+function makeRequestWithFormdata(requestParams, callback) {
```

Comment:
Should this be `FormData`?

Reply:
They aren't all formdata, I wrapped all the params in one object to not have too many in the call signature

Reply:
Oh, I literally meant the camelCase naming should probably be `makeRequestWithFormData `